README.md

```diff
@@ -1 +1,26 @@
-# proposals
\ No newline at end of file
+# HH.js Talk Proposals
+
+If you would like to give a talk at [HH.js](https://www.meetup.com/hamburg-js/), thank you very much! We really appreciate your willingness to put your time and effort into something that will make our meetup more interesting and inspiring.
+
+## Proposals
+
+Please [open an issue](https://github.com/hamburg-js/proposals/issues/new) in this repository. In that issue, we ask you to outline the topic of your talk and to pick a couple of possible dates. Oh, and please let us know if you have already given it before.
```

Comment:
It'd would imho make sense to add `?template=talk-proposal.md` to the "New issue"-URL and to merge PR #1 first then.